alibi/explainers/similarity/backends/__init__.py

```diff
@@ -0,0 +1,21 @@
+from alibi.utils.frameworks import has_pytorch, has_tensorflow
+
+if has_pytorch:
+    # import pytorch backend
+    from alibi.explainers.similarity.backends.pytorch.base import _TorchBackend
+
+if has_tensorflow:
+    # import tensorflow backend
+    from alibi.explainers.similarity.backends.tensorflow.base import _TensorFlowBackend
+
+
+def _select_backend(backend, **kwargs):
```

Comment:
Add typing to `backend` argument. Maybe consider `Literal` instead of simple `str`.

Reply:
Since it's a private function we can use the `Framework` enum defined in `utils.frameworks`.